### DISCUSSION TITLE
【keyball44】強く押し込んだり指で挟むと入力できる

### DISCUSSION THREAD

**Discussion by rin419** [Yowkees/keyball#615] [Q&A]
初めまして。
keyball44で、画像矢印の指すひとつのキーだけが上手く入力できず困っております。
![IMG_1714](https://github.com/Yowkees/keyball/assets/173567263/4b5f7785-6217-4cfa-a39b-de945b409e2c)
・ピンセットでショートさせると入力される
・キーを強く押し込んだり、親指と人差し指で上下を挟むようにしてキーを押すと入力できる
・一度入力できたらそのまま軽いタッチで入力し続けることができる
・一度ケーブルを抜いたりするとリセットされるのか？抜いた衝撃で物理的に動くのか？また入力できなくなる
・他の問題ないキースイッチと入れ替えても状況に変化なし
といった状態です。

素人なりに接触が悪い状態なのかなと疑ってはみますが、そこまではんだが悪い状態にも思えず、かえって無駄に焦がしたりしそうで躊躇しております。
ここがマズそうですとか、ここをチェックしてみろといったアドバイスがありましたらいただけると助かります。
よろしくお願いします。

**Comment by AK-yonnana** [comment]
![スクリーンショット 2024-06-23 172602](https://github.com/Yowkees/keyball/assets/77046594/ddc31c55-8102-43f6-a809-b6a3cd9dca95)
緑矢印の2か所がキースイッチと、ソケットの間の接触点です。スイッチ側の端子とソケットの端子の接触が甘いのかもしれません。ソケット側の端子をピンセットなどで起こすなどして、接触を改善したらよいかもしれません。参考まで。

**Reply by rin419** [reply]
ありがとうございます！
なるほどキーソケットの端子…と思い周囲のソケットと見比べてみると、確かに隙間が大きいかも…？と思いましたので、いっそのことと思い、予備のキーソケットと交換して付け直してみました。
しかし、状況は変わらず…
・付け直し後もショートさせると入力されることは確認済み
・指で挟むと前はすぐに入力開始できていたが、付け直してからはちょっと反応が悪くなったような気がする（何度か押していると入力できるようにはなる）
・一度入力できるようになれば軽いタッチでできるのも変わらず

他に何か原因になりそうなポイントがありますでしょうか？
このキーひとつだけとはいえかなり使うキーでもあるので、試せることは何でも試してみたいと思っています。
よろしくお願いします。

**Reply by Ryo624** [reply]
ピンセットでの入力は問題ないとのことですので、おそらくキーソケット自体のはんだ付けが原因かと思われます。
ソケットの両端が端子にしっかり馴染むように根気よくはんだ付けをやり直してみてください。

**Reply by rin419** [reply]
ありがとうございます！解決しました！
おっしゃるとおりソケット周りのはんだを念入りにやり直しては試し、やり直しては試し…をひたすら繰り返しましたら、他のキーと同様に軽いタッチで入力できるようになりました。
原因特定に自信がなく、少し試して改善しなければ違うか…と他を探すようなことを繰り返していましたので、お2人に絞り込んでいただいたおかげで解決できました。
お忙しい中どうもありがとうございました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます。
ぜひ末永くご愛用いただけましたら幸いです！
